traits/etsconfig/etsconfig.py

```diff
@@ -314,7 +314,7 @@ def _get_kiva_backend(self):
                     self._kiva_backend = (
                         "quartz" if sys.platform == "darwin" else "image"
                     )
-                elif self.toolkit == "qt4":
+                elif self.toolkit.startswith("qt"):
```

Comment:
Could we stick to a whitelist of toolkits we allow? (Probably just `"qt4"` and `"qt"`.) Otherwise we risk having to support everything starting with "qt" for evermore.

Reply:
(Though I guess it's a moot point, because the fall-through branch will pick `"image"` anyway.)